pkg/kubelet/pod_workers.go

```diff
@@ -664,6 +684,18 @@ func (p *podWorkers) UpdatePod(options UpdatePodOptions) {
 	}
 }
 
+// isConflictedPod returns true when there is a known pod with the same
+// fullname as pod
+func (p *podWorkers) isConflictedPod(pod *v1.Pod) bool {
```

Comment:
```suggestion
func (p *podWorkers) isConflictingStaticPod(pod *v1.Pod) bool {
```

Lets help future folks and reduce the scope of the name a little